src/sherpa_ai/utils.py

```diff
@@ -48,7 +48,60 @@ def load_files(files: List[str]) -> List[Document]:
             documents.extend(loader.load())
     logger.info(documents)
     return documents
+import json
+from typing import Dict, List
 
+class JsonToObject:
```

Comment:
Since these utils are specific to the prompt, can we move it to the `prompts` module?